Tools/simulation/gz/models/rover/model.sdf

```diff
@@ -0,0 +1,388 @@
+<sdf version="1.8">
+  <model name="rover">
+    <pose>0 0 .04 0 0 0</pose>
+    <self_collide>false</self_collide>
+    <static>false</static>
+    <link name="rover/Base">
```

Comment:
Currently, the imu sensor is not getting added to the simulation. 'rover/Base' has to be renamed to 'base_link'.